### ISSUE TITLE
Feat: Support for Safari browser

### ISSUE THREAD

**Issue by ShivamJoker** [hoppscotch/hoppscotch-extension#100]
A lot people use Safari too, it would be great to have the support for it as well.

I have some experience in building extension with Safari so I may give it a shot.

The messaging API is supported as well

**Comment by AndrewBastin** [comment]
Yup. Totally agree. Pretty sure the extension should work right out of the box (or atleast hope so, because Safari). The only reason I don't have it officially in Safari is because I literally don't have a Mac to test it on 🤣

Hopefully eventually I will be getting one and I will get it official with an App Store listing and everything.

(also ticket no. 100!)

**Comment by ShivamJoker** [comment]
I have got both it will need some changes I'll give it a shot and create PR

**Comment by AndrewBastin** [comment]
Awesome!

But then again, I am not thinking about announcing official support until I get a Mac to work on this which I am planning get one in December.

**Comment by felix-schindler** [comment]
Any news on this issue?
